src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sat calendar--month-length-29">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--sat calendar--month-length--29">
```

pay attention to modifiers naming convention